index.html

```diff
@@ -47,5 +47,189 @@ <h3>LET'S CONNECT</h3>
+              </div>
+           </div> 
+         </div>
+         
+         <div class="works-flex">
+           <div class="works-flex-item work-image">
+             <img src="snapshoot_portfolio(1).png" class="img-style" alt="tonic works" />
+           </div>
+           <div class="works-flex-item">
+             <h3 class="intro-works-text">Multi-Post <br/> Stories</h3>
+           </div>
+           <ul class="works-flex=item titles-container">
+             <li class="list-main">CANOPY</li>
+             <li class="titles-item">Back End Dev</li>
+             <li class="titles-item">2015</li>
+           </ul>
+           <p class="works-flex-item works-description">
+             A daily selection of privately personalized reads; no accounts or sign-ups required.
+           </p>
+           <div class="works-flex-item">
+             <ul class="lang-container">
+               <li class="lang">html</li>
+               <li class="lang">css</li>
+               <li class="lang">javascript</li>
+             </ul>
+           </div>
+           <div class="works-flex-item">
+            <div class="btn-see">
+               <a href="#" class="btn-see-project">See project</a>
+            </div>
+           </div> 
+         </div>
+
+         <div class="works-flex">
+           <div class="works-flex-item work-image">
+             <img src="snapshoot_portfolio(2).png" class="img-style" alt="tonic works" />
+           </div>
+           <div class="works-flex-item">
+             <h3 class="intro-works-text">Tonic</h3>
+           </div>
+           <ul class="works-flex-item titles-container">
+             <li class="list-main">CANOPY</li>
+             <li class="titles-item">Back End Dev</li>
+             <li class="titles-item">2015</li>
+           </ul>
+           <p class="works-flex-item works-description">
+             A daily selection of privately personalized reads; no accounts or sign-ups required.
+           </p>
+           <div class="works-flex-item">
+             <ul class="lang-container">
+               <li class="lang">html</li>
+               <li class="lang">css</li>
+               <li class="lang">javascript</li>
+             </ul>
+           </div>
+           <div class="works-flex-item">
+            <div class="btn-see">
+               <a href="#" class="btn-see-project">See project</a>
+             </div>
+           </div>
+         </div>
+         <div class="works-flex">
+           <div class="works-flex-item work-image">
+             <img src="snapshoot_portfolio(3).png" class="img-style" alt="tonic works" /> 
+           </div>
+           <div class="works-flex-item">
+             <h3 class="intro-works-text">Multi-Post <br/> Stories</h3>
+           </div>
+           <ul class="works-flex-item titles-container">
+             <li class="list-main">CANOPY</li>
+             <li class="titles-item">Back End Dev</li>
+             <li class="titles-item">2015</li>
+           </ul>
```

Comment:
- [ ] There is an issue with this details list, because it's being shown in the middle of the card, and according to the Figma design it should be aligned to the left. I think you could use `align-self` instead of `justify-self` :wink:

  Your design:
  ![image](https://user-images.githubusercontent.com/81522435/143592817-22024dd2-0c32-40d0-a566-71b22c7c048e.png)

  Figma design:
  ![image](https://user-images.githubusercontent.com/81522435/143592948-b4f82a62-851e-4d78-9b9d-51038142ef95.png)